frontend/ui/elements/font_settings.lua

```diff
@@ -54,6 +54,21 @@ function FontSettings:getPath()
     return getUserDir()
 end
 
+function FontSettings:getAndroidPath()
+    local A, android = pcall(require, "android")
+    local system_path = "/system/fonts"
+    local user_path = "/koreader/fonts"
+    local legacy_font_dirs = system_path..";".."/sdcard"..user_path
+
+    if not A then return legacy_font_dirs end
```

Comment:
Bit confused by this? For easier testing (in the emulator)?

Reply:
That is, /system/fonts or /sdcard/koreader/fonts won't be very useful on non-Android platforms regardless.

Reply:
yeah, more than anything a future proof thingy. I guess nobody will want to call FontSettings:getAndroidPath() outside of android. But if someone tries that it will receive the "legacy" android path.

Not sure if this is a good idea but is safer in the case we cannot load the android module.

Reply:
Why not just nil or an empty string?